docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -0,0 +1,177 @@
+---
+title: View COPY status with pg_stat_progress_copy
+linkTitle: View COPY status
+description: Use pg_stat_progress_copy to get the COPY command status, number of tuples processed, and other COPY progress reports.
+headerTitle: View COPY status with pg_stat_progress_copy
+image: /images/section_icons/index/develop.png
+menu:
+  preview:
+    identifier: pg-stat-progress-copy
+    parent: query-tuning
+    weight: 400
+type: docs
+---
+
+Yugabyte uses the PostgreSQL `pg_stat_progress_copy` view to report the progress of the COPY command execution. Whenever COPY is running, the `pg_stat_progress_copy` view will contain one row for each backend that is currently running a COPY command.
+
+The following table includes the columns and their description:
+
+| Column | Description |
+| :---- | :---------- |
+| pid | Process ID of backend. |
+| datid | OID of the database to which this backend is connected. |
+| datname | Name of the database to which this backend is connected. |
+| relid | OID of the table on which the COPY command is executed. It is set to 0 if copying from a SELECT query.|
+| command | The command that is running COPY FROM, or COPY TO. |
+| type | The I/O type that the data is read from or written to: FILE, PROGRAM, PIPE (for COPY FROM STDIN and COPY TO STDOUT), or CALLBACK (used for example during the initial table synchronization in logical replication). |
+| yb_status | Tracking status of the copy command. |
+| bytes_processed | Number of bytes already processed by COPY command. |
+| bytes_total | Size of the source file for COPY FROM command in bytes. It is set to 0 if not available. |
+| tuples_processed | Number of tuples already processed by COPY command. |
+| tuples_excluded | Number of tuples not processed because they were excluded by the WHERE clause of the COPY command. |
+
+## YugabyteDB specific changes
+
+The `pg_stat_progress_copy` view includes the following changes which are YugabyteDB specific:
+
+### Definition of `tuples_processed`
+
+The definition of `tuples_processed` column is different in YugabyteDB in comparison to PostgreSQL.
+
+For the `COPY` command in YugabyteDB, an option `ROWS_PER_TRANSACTION` is added which defines the transaction size to be used by the `COPY` command. For example, if the total tuples to be copied are 5000 and `ROWS_PER_TRANSACTION` is set to 1000, then the database will create 5 transactions and each transaction will insert 1000 rows. If there is an error during the execution of the copy command, then some tuples can be persisted based on the already completed transaction. This also implies that if the error occurs after inserting the 3500th row, then the first 3000 rows will still be persisted in the database.
```

Comment:
It is better to add further description in https://docs.yugabyte.com/preview/api/ysql/the-sql-language/statements/cmd_copy/  for the `ROWS_PER_TRANSACTION`  and link the page here. 

Also mention the default for ROWS_PER_TRANSACTION as well. 

IN 2.14/2.15  default is 20K and older releases as default as 1000. 

Finally we can on this page we can mention the high level summary with example. Duplicating  some text is also fine.

Reply:
@sushantrmishra , Thanks for all your suggestions. I have addressed them and if could you please look at this one in particular? I have included the details on both the pages and have linked them as well.